webhook_to_fedora_messaging/endpoints/models/message.py

```diff
@@ -0,0 +1,33 @@
+from abc import ABC
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class MessageBase(BaseModel, ABC):
+    """
+    Base: Message
+    """
+
+    class Config:
+        from_attributes = True
+
+
+class MessageExternal(MessageBase):
+    data: Optional[str]
+
+
+class MessageRequestMain(BaseModel):
+    body: Optional[str]
+
+
+class MessageRequest(BaseModel):
+    data: MessageRequestMain
```

Comment:
I don't think we can expect the incoming webhook to have any format we can describe here, and certainly not to put its payload in a `data` key (at least Github does not do it).
I think we can't validate incoming webhooks with pydantic, so the data type should just be `dict`. If you agree, let's remove this class and use `dict` as the type in the endpoint.